### ISSUE TITLE
gnt-cluster copyfile fails when the path to the file isn't absolute

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#810]
Originally reported of Google Code with ID 754.

```
gnt-cluster version

Software version: 2.9.3
Internode protocol: 2090000
Configuration format: 2090000
OS api version: 20
Export interface: 0
VCS version: v2.9.3

hspace --version

hspace (ganeti) version v2.9.3
compiled with ghc 7.4
running on linux x86_64

Release:

DISTRIB_RELEASE="7 (wheezy) - installer build 20130613+deb7u1+b1"


<b>What steps will reproduce the problem?</b>

1. gnt-cluster copyfile file_in_current_directory

What is the expected output? 

gnt-cluster copyfile should distribute the file to all nodes in the cluster.

What do you see instead?

Copy of file file_in_current_directory to node XXX failed.

Please provide any additional information below:

The manual page for gnt-cluster, section 'copyfile' uses absolute paths as an example, but doesn't state that
this is a requirement.

Proposed change:

- better error message (test that path is absolute) from the `gnt-cluster copyfile` command, possibly
  'copyfile requires an absolute (qualified) path'

- add this requirement in the manpage:

'Note that {file} must be an absolute path - failure to provide an absolute path (with leading /) will result in an error'
```
Originally added on 2014-03-07 16:56:27 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2014-03-07 17:01:51 +0000 UTC.

Added Labels: Type-Documentation Priority-Low
Changed State: Accepted
Added to Milestone: Release2.9

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 74a4fc947681e88493d19f31ca4adfcfffbf5481
Author: Klaus Aehlig <aehlig@google.com>
Date:   Fri Mar 7 18:11:28 2014 +0100

    gnt-cluster copyfile: accept relative paths
    
    If, on the command line, the argument to gnt-cluster copyfile is
    a relative path, consider this a shorthand for the corresponding
    absolute path.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Michele Tartara <mtartara@google.com>

```
Originally added on 2014-03-07 17:26:48 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Documentation
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.10.2.
```
Originally added on 2014-03-24 12:18:18 +0000 UTC.

Changed State: Released
